src/main/java/com/mraof/minestuck/alchemy/PredeterminedCardCaptchas.java

```diff
@@ -0,0 +1,21 @@
+package com.mraof.minestuck.alchemy;
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.HashBiMap;
+import net.minecraft.world.item.Item;
+
+
+public class PredeterminedCardCaptchas
+{
+	private static BiMap<Item, String> predefinedCardMap = HashBiMap.create();
+	
+	public static void SetData(BiMap<Item, String> predefinedCards)
```

Comment:
```suggestion
	static void SetData(BiMap<Item, String> predefinedCards)
```
Reduced visibility because it should only be accessed from `PredeterminedCaptchaLoader`.
Although if you turn `PredeterminedCaptchaLoader` into an inner class of `PredeterminedCardCaptchas` (similar to `ComputerThemes.Loader` or `DialogueNodes.Loader`), then you can even use private access to set it.